api/views/maps.py

```diff
@@ -0,0 +1,77 @@
+from api import app
+from flask import Blueprint, request
+from api import db
+from api.models import Map
+import json
+from flask import jsonify
+#from api.utils import InvalidUsage
+from sqlalchemy import func
+# import time
+# from datetime import date
+# import uuid
+from flask_sqlalchemy import SQLAlchemy
+from api.utils import create_response
+
+
```

Comment:
since im not sure if you saw this in the previous PR, make sure to follow all the [Flask API Conventions](https://github.com/hack4impact-uiuc/wiki/wiki/Flask-API-Conventions)